authorization/service/getUserDevices.go

```diff
@@ -42,48 +41,25 @@ func sendUserDevices(request *pb.GetUserDevicesRequest, srv pb.AuthorizationServ
 	}
 
 	for _, deviceID := range ids {
-		err := srv.Send(&pb.UserDevice{DeviceId: deviceID, UserId: d.Owner})
+		err := srv.Send(&pb.OwnerDevice{DeviceId: deviceID})
 		if err != nil {
 			return log.LogAndReturnError(status.Errorf(status.Convert(err).Code(), "cannot get user devices: %v", err))
 		}
 	}
 	return nil
 }
 
-// GetUserDevices returns a list of user's devices if the access token is valid.
-func (s *Service) GetUserDevices(request *pb.GetUserDevicesRequest, srv pb.AuthorizationService_GetUserDevicesServer) error {
+// GetOwnerDevices returns a list of user's devices if the access token is valid.
+func (s *Service) GetOwnerDevices(request *pb.GetOwnerDevicesRequest, srv pb.AuthorizationService_GetOwnerDevicesServer) error {
```

Comment:
You didn't rename the file. But let's discuss if it's GetOwnerDevices or GetDevices.

Reply:
`Owner` prefix removed types and functions; file renamed.